generators/client/templates/react/src/main/webapp/app/shared/layout/header/menus/admin.tsx.ejs

```diff
@@ -28,35 +28,35 @@ import { Translate, translate } from 'react-jhipster';
 const adminMenuItems = (
   <>
   <%_ if (applicationType === 'gateway' && serviceDiscoveryType) { _%>
-    <DropdownItem tag={Link} to="/admin/gateway"><FontAwesomeIcon icon="road" /> <Translate contentKey="global.menu.admin.gateway" /></DropdownItem>
+    <DropdownItem tag={Link} to="/admin/gateway"><FontAwesomeIcon icon="road" /> <Translate contentKey="global.menu.admin.gateway">Gateway</Translate></DropdownItem>
```

Comment:
Strange, I'm pretty sure this was working fine before, maybe something changed. The stripI18n process will fetch content of the EN i18n file and replace the Translate component here

Reply:
It's not working for the header. The issue has been there for a while but I forgot about it because of the yarn problem we have.
I know we're removing tags here 
https://github.com/jhipster/generator-jhipster/blob/bed37de5d39a563bf1d21f7223581c4d6eafb8a7/generators/generator-base.js#L1480 
but where do we replace it with the actual translation? 'Cause I think this part is not working at all atm.

Reply:
Oh sorry you are right the replacement only works for placeholders https://github.com/jhipster/generator-jhipster/blob/bed37de5d39a563bf1d21f7223581c4d6eafb8a7/generators/utils.js#L254

Reply:
but somehow this was working I guess otherwise this would be pretty obvious for me to not notice when doing the i18n part as its on the menu

Reply:
Yes, as far as I can remember it was working but I wasn't sure when I placed the placeholders. I'll try to fix that today.